src/doc/gui_manual/BuildingOnMac/Execution.rst

```diff
@@ -27,5 +27,8 @@ Once masonry is running, it will produce log files in the ``_logs`` directory. T
 
    Mansonry Logs in HTML format
 
-
+Note that if you already have a web browser running on your system, that
+the web browser may not be able to connect to the local web server created
```

Comment:
Should it be 'that web browser' instead of 'that the web browser'?

Reply:
I updated the wording.